src/modules/ROOT/pages/changelog/202303/01.json

```diff
@@ -0,0 +1,889 @@
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/6ade04484397285d541a2da1ff88e497c98717fb" 
+      ]
+    },
+    {
+      "what": [
+        "the overlapping between zoom button on the geometry page block with the page block edit group buttons."     
+      ],
+      "why": [
+        "to ensure the page block edit group button is clickable on geometry page blocks."
+      ],
+      "how": [
+        "by updating the styling option (z-index) for the page block edit to overlap the zoom edit buttons."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/821/commits/5717d8e480a4ec96f256c420cc4b2228b28c1e9c" 
+      ]
+    },
+    {
+      "what": [
+        "module validation rules for unique value constraints previously used as validation for record duplication detection."
+      ],
+      "why": [
+        "to extend support for record duplicate detection with multiple fields in single rule with various detection algorithm and muli-value matching criteria."
+      ],
+      "how": [
+        "by extending record duplicate detection configuration to support multi value matching criteria and value checking algorithms."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/617/commits/5f4d02ac8400ec2963ffba8bfcb19a13de1ebb5a",
+        "https://github.com/cortezaproject/corteza/pull/617/commits/e7fa07cfae7d66f92da933f7c4666a2d8f30fd4e"
+      ]
+    },
+    {
+      "what": [
+        "uninformative internal error error message on the user profile change password interface."
+      ],
+      "why": [
+        "to help users understand better of what and why the error was caused."
+      ],
+      "how": [
+        "by properly handling and serving the error message."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/73a29b6b9"
+      ]
+    },
+    {
+      "what": [
+        "the usage of chart.js on admin dashboard with e-charts."
+      ],
+      "why": [
+        "to remove all places where we used to use chart.js since we transitioned to apache e-charts."
+      ],
+      "how": [
+        "The chart is now in line with other charts in {PRODUCT_NAME}, the date tooltips are now also correctly displayed."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/390a14ea73b7e0836fccd735a648baa3c42b730d"
+      ]
+    },
+    {
+      "what": [
+        "the corresponding marker when opening a specific geometry location, including on geometry multivalue fields.",
+        "Before, the map was opening on the same location as specified in the module field, not taking into account the marker location.",
+        "Obscure other map markers (if present) with some opacity to make the current marker stand out in a multivalue field.",
+        "Zoom in to a more appropriate level.",
+        "Before, the map markers were too far off and users had to zoom in all the time to see the exact coordinates."
```

Comment:
Can we try to shorten and condense this blob? I think it's too verbose for the changelog